tst/PortingAssistantExtensionUITests/VisualStudioSession.cs

```diff
@@ -20,11 +20,12 @@ namespace PortingAssistantExtensionUITests
     public class VisualStudioSession
     {
         protected const string WindowsApplicationDriverUrl = "http://127.0.0.1:4723";
-        private const string VSAppId = @"C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\Common7\IDE\devenv.exe";
+        private const string VS2019AppId = @"C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\Common7\IDE\devenv.exe";
         protected const string winAppDriverExe = "C:\\Program Files (x86)\\Windows Application Driver\\WinAppDriver.exe";
-        private const string testSolutionsDir = "C:\\ide-ui-test-solutions";
-        private const string testSolutionsZip = "C:\\ide-ui-test-solutions.zip";
-        private static bool firstTimeSetupRequired = true;
+        protected const string testSolutionsDir = "D:\\ide-ui-test-solutions";
```

Comment:
I think its better not to change the location, this may cause issue when we run it on the Ec2